### DISCUSSION TITLE
adwaita

### DISCUSSION THREAD

**Discussion by aquifolly** [catppuccin/catppuccin#2511] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://github.com/aquifolly/catppuccin-adwaita

### Any additional comments?

this is an alternative route using css directly, and this also depends upon adw-gtk3

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by ghost** [comment]
+1 for css directly. Who needs a framework for CSS styling?

I'm also contributing to a different gtk project that will have some robust build and test tools that I can plop into this project sometime around December–Jan.

Maybe we can have tests that show us how different popular or problematic apps look?

**Comment by soymadip** [comment]
so will gtk be available?

**Reply by gradylink** [reply]
GTK was available but was removed because of how hard it was to maintain, it is now recommended to use https://github.com/vinceliuice/Colloid-gtk-theme instead (the theme the catppuccin/gtk was based off of.)

**Reply by soymadip** [reply]
ok

**Comment by musjj** [comment]
There's also [catppuccin-adw](https://github.com/claymorwan/catppuccin/tree/main/adw) which was created using the [adw-colors](https://github.com/lassekongo83/adw-colors) tool.

I personally think this approach is the easiest to maintain. Any thoughts?

**Comment by LuminarLeaf** [comment]
https://github.com/LuminarLeaf/adw-catppuccin

Shameless plug; A soft fork of claymorwan's original repo, but adds separate adw-gtk3 theming as well and also adds some things from other themes at https://github.com/lassekongo83/adw-colors/tree/main/themes.

Open to fix stuff to match the org guidelines if they want to add it. <3
